### ISSUE TITLE
TypeError: compile_checkpoint() takes 2 positional arguments but 3 were given

### ISSUE THREAD

**Issue by YanWittmann** [d8ahazard/sd_dreambooth_extension#172]
**Describe the bug**
When using the 'Generate ckpt' button, the error message `TypeError: compile_checkpoint() takes 2 positional arguments but 3 were given` appears. I have the trained model selected in the drop down above and the latest version installed.

![Screenshot 2022-11-13 220253](https://user-images.githubusercontent.com/37689635/201544447-70e51d7f-a9fa-4ba0-a58d-c6a1d904b533.png)

**Provide logs**
```
Error loading emedding test:
Traceback (most recent call last):
  File "E:\Users\myuser\files\programming\stable-diffusion-all\stable-diffusion-webui\modules\textual_inversion\textual_inversion.py", line 133, in load_textual_inversion_embeddings
    process_file(fullfn, fn)
  File "E:\Users\myuser\files\programming\stable-diffusion-all\stable-diffusion-webui\modules\textual_inversion\textual_inversion.py", line 103, in process_file
    if 'string_to_param' in data:
TypeError: argument of type 'NoneType' is not iterable

Loaded a total of 0 textual inversion embeddings.
Embeddings:
Running on local URL:  http://0.0.0.0:7860

To create a public link, set `share=True` in `launch()`.
Traceback (most recent call last):
  File "E:\Users\myuser\files\programming\stable-diffusion-all\stable-diffusion-webui\venv\lib\site-packages\gradio\routes.py", line 284, in run_predict
    output = await app.blocks.process_api(
  File "E:\Users\myuser\files\programming\stable-diffusion-all\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 982, in process_api
    result = await self.call_function(fn_index, inputs, iterator)
  File "E:\Users\myuser\files\programming\stable-diffusion-all\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 824, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "E:\Users\myuser\files\programming\stable-diffusion-all\stable-diffusion-webui\venv\lib\site-packages\anyio\to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "E:\Users\myuser\files\programming\stable-diffusion-all\stable-diffusion-webui\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "E:\Users\myuser\files\programming\stable-diffusion-all\stable-diffusion-webui\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 867, in run
    result = context.run(func, *args)
TypeError: compile_checkpoint() takes 2 positional arguments but 3 were given
```

**Environment**
Native Windows using NVIDIA GeForce RTX 3090

**Comment by noonz66** [comment]
Temp Solution:
In main.py of dreambooth extension folder
Look for this piece of code:

db_generate_checkpoint.click(
            fn=conversion.compile_checkpoint,
            inputs=[
                db_pretrained_model_name_or_path,
                db_pretrained_vae_name_or_path,
                db_mixed_precision
            ],
            outputs=[
                db_progress,
                db_outcome
            ]
        )

and comment db_pretrained_vae_name_or_path
to look like:

  db_generate_checkpoint.click(
            fn=conversion.compile_checkpoint,
            inputs=[
                db_pretrained_model_name_or_path,
               # db_pretrained_vae_name_or_path,
                db_mixed_precision
            ],
            outputs=[
                db_progress,
                db_outcome
            ]
        )

make sure to restart the whole SD

**Comment by YanWittmann** [comment]
Alright, thank you, it now works again!

**Comment by d8ahazard** [comment]
Fixed with f946126
